<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry"><rect><x>0</x><y>0</y><width>300</width><height>200</height></rect></property>
  <property name="windowTitle"><string>Login</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item><widget class="QComboBox" name="roleComboBox">
     <item><property name="text"><string>admin</string></property></item>
     <item><property name="text"><string>operator</string></property></item>
   </widget></item>
   <item><widget class="QLineEdit" name="usernameLineEdit"><property name="placeholderText"><string>Username</string></property></widget></item>
   <item><widget class="QLineEdit" name="passwordLineEdit"><property name="echoMode"><enum>QLineEdit::Password</enum></property><property name="placeholderText"><string>Password</string></property></widget></item>
   <item><widget class="QPushButton" name="loginButton"><property name="text"><string>Login</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
